charms/katib-ui/src/charm.py

```diff
@@ -137,12 +119,20 @@ def _get_interfaces(self):
             raise CheckFailed(err, BlockedStatus)
         return interfaces
 
-    def _check_image_details(self):
-        try:
-            image_details = self.image.fetch()
-        except OCIImageResourceError as e:
-            raise CheckFailed(f"{e.status.message}", e.status_type)
-        return image_details
+    # def _check_image_details(self):
+    #     try:
+    #         image_details = self.image.fetch()
+    #     except OCIImageResourceError as e:
+    #         raise CheckFailed(f"{e.status.message}", e.status_type)
+    #     return image_details
```

Comment:
Instead of commenting code, its best practice to just remove it entirely, git will keep the code history in case we want to consult it in the future.